src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { useState } from "react";
+
+import { Icon } from "@/components/Icon/Icon";
+
+type PopoverProps = {
+  children: React.ReactNode;
+};
+export const Popover: React.FC<PopoverProps> = (props) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const handlePopOverOpen = () => setIsPopoverOpen(!isPopoverOpen);
+
+  return (
+    <div className="relative">
+      <button type="button" aria-label="Toggle Menu" onClick={handlePopOverOpen}>
+        <Icon type="DotsCircleHorizontal" color="black" />
+      </button>
+      <ul
+        className={`absolute right-3 top-10 rounded bg-white shadow drop-shadow ${isPopoverOpen ? "block" : "hidden"}`}
```

Comment:
こっちの方がFigmaに忠実で綺麗な気がするのですがどうでしょう？
```suggestion
        className={`absolute right-0 top-7 rounded bg-white shadow drop-shadow ${isPopoverOpen ? "block" : "hidden"}`}
```